chore: updated dependencies (aligned again with nextjs-auth0)

natterstefan opened the pull request:
## Task

Hi @sir-dunxalot, I am about to resolve some of the open issues and as a first step, I updated the dependencies to the latest supported (and required) version.

Let me know what you think about it, please.

I will continue with my suggestions to help resolve some issues of others and the ones I experienced.

sir-dunxalot commented:
Hey @natterstefan. I appreciate the effort here! All sounds good to me.

Personally, I use yarn and that is what I built this project with so yarn is fine :)

Let me know when this are ready for review and merge (or if you have any questions) and we'll get PRs merged in.

natterstefan commented:
Hi @sir-dunxalot,

thanks for the quick reply, appreciated it! I must have missed the existing `yarn.lock` file 😓 . I removed the `package-lock.json` and updated `yarn.lock` instead.

**This PR is ready for review now.**

---

I have one question related to the suggestions I'll open via PRs soonish. Are you up for replacing Travis with GitHub actions? It's an open-source project which means GitHub Actions are free: https://github.com/pricing

sir-dunxalot commented:
@natterstefan I've pulled down this branch. Everything is working (wahoo!) except the test suite fails with 404s for API routes.

![image](https://user-images.githubusercontent.com/4495352/153026440-743cad4d-d795-402c-9bb5-8cd118267212.png)

Are you seeing the same issue? Maybe one of the dep updates requires a code change?

Let me know if you need a `cypress.env.json` file for local testing and I can send you one via email, Discord, Slack, etc.

natterstefan commented:
Interesting 🤔 All tests pass for my created example app for this. Sure let's figure out what's going on together. I'll send you an email now.

natterstefan commented:
@sir-dunxalot FYI, I already created a follow-up PR for this one where I refactored the ENV variable handling and moved from Travis to GitHub Actions (because every fork can use GitHub Actions without setting up an additional Travis account then; and personally, I prefer GitHub Actions these days. 😅)
